packages/node_modules/@webex/webex-core/src/lib/credentials/credentials.js

```diff
@@ -130,6 +130,34 @@ const Credentials = WebexPlugin.extend({
     /* eslint-enable camelcase */
   },
 
+  /**
+   * Get the determined OrgId.
+   *
+   * @throws {Error} - If the OrgId could not be determined.
+   * @returns {string} - The OrgId.
+   */
+  getOrgId() {
+    this.logger.info(
+      'credentials: attempting to retrieve the OrgId from token'
+    );
+
+    try {
+      // Attempt to extract a client-authenticated token's OrgId.
+      this.logger.info('credentials: trying to extract OrgId from JWT');
+
+      return this.extractOrgIdFromJWT(this.supertoken.access_token);
+    }
+    catch (e) {
+      // Attempt to extract a user token's OrgId.
+      this.logger.info('credentials: could not extract OrgId from JWT');
+      this.logger.info(
+        'credentials: attempting to extract OrgId from user token'
```

Comment:
Is this in a different line because of line-length limits?

Reply:
That is correct.